packages/passport/sdk/package.json

```diff
@@ -22,6 +22,7 @@
     "ethers": "^5.7.2",
     "jwt-decode": "^3.1.2",
     "magic-sdk": "^21.2.0",
+    "mipd": "^0.0.5",
```

Comment:
This will also need to be included as a dependency in `packages/passport/sdk/package.json`